plutus-pab/app/Cli.hs

```diff
@@ -201,15 +202,15 @@ runConfigCommand t _ Config {nodeServerConfig, chainIndexConfig} serviceAvailabi
         serviceAvailability
 
 -- Install a contract
-runConfigCommand t _ Config{dbConfig} _ (InstallContract contractExe) = do
-    connection <- App.dbConnect (LM.convertLog LM.PABMsg t) dbConfig
+runConfigCommand t _ Config{dbConfig} _ _ (InstallContract contractExe) = do
+    connection <- Left <$> App.dbConnect (LM.convertLog LM.PABMsg t) dbConfig
```

Comment:
Sorry, i don't quite follow here - why do we have to fmap Left over the result of `dbConnect` here?